storybook/stories/scales/8_linear_binary.story.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { boolean, select } from '@storybook/addon-knobs';
+import React from 'react';
+
+import { Axis, Chart, LineSeries, Position, ScaleType, Settings } from '@elastic/charts';
+import { getRandomNumberGenerator } from '@elastic/charts/src/mocks/utils';
+
+import { useBaseTheme } from '../../use_base_theme';
+
+const rng = getRandomNumberGenerator();
+
+function randomByte() {
+  return rng(0, 1);
+}
+
+function generateRandomBinary(binaryLength: number) {
+  let binary = '0b';
+  for (let i = 0; i < binaryLength; ++i) {
+    binary += randomByte();
+  }
+  return binary;
+}
```

Comment:
```suggestion
function generateRandomBinary(bitCount: number) {
  const number = rng(0, 2 ** bitCount);
  return `0b${number.toString(2).padStart(bitCount, '0')}`;
}
```
It was `randomBit` not `randomByte` though this single expression removes the need

Reply:
Btw. I don't fully understand the purpose, are we now expressly allowing strings into our API for numbers? Or is it a preexisting capability? I don't know why a user of the library wouldn't just pass a numeric value `rng(0, 2 ** bitCount)` instead of a string, are there some fractional accuracy concerns?

Reply:
Good idea. Changed in 9d25939